### PR TITLE
WAVE2-848-adding fields and template for admissions deadline section

### PR DESCRIPTION
**Pull request by nindusaini**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 1]
```diff
@@ -95,3 +95,53 @@
...
+            foreach ($fc_item->field_fgs_ad_deadline_details_ca[LANGUAGE_NONE] as $key_ca => $value_ca) {
+                $fc_item_ca = field_collection_item_load($value_ca['value']);
+                if(isset($fc_item_ca->field_fgs_ad_deadline_ca_date[LANGUAGE_NONE][0]['value'])){
+                    //convert the date to desired format
+                    $date_ca = date("F j, Y" , strtotime($fc_item_ca->field_fgs_ad_deadline_ca_date[LANGUAGE_NONE][0]['value']));
+                }
+                if(isset($fc_item_ca->field_fgs_ad_deadline_ca_desc[LANGUAGE_NONE][0]['value'])){
+                    $desc_ca = $fc_item_ca->field_fgs_ad_deadline_ca_desc[LANGUAGE_NONE][0]['value'];
+                }
+                echo '<p>' . $date_ca . ' : ' . $desc_ca . '</p>';
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Add commenting - what is being printed out here?

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 2]
```diff
@@ -95,3 +95,53 @@
...
+            foreach ($fc_item->field_fgs_ad_deadline_details_in[LANGUAGE_NONE] as $key_in => $value_in) {
+                $fc_item_in = field_collection_item_load($value_in['value']);
+                if(isset($fc_item_in->field_fgs_ad_deadline_in_date[LANGUAGE_NONE][0]['value'])){
+                    //convert the date to desired format
+                    $date_in = date("F j, Y" , strtotime($fc_item_in->field_fgs_ad_deadline_in_date[LANGUAGE_NONE][0]['value']));
+                }
+                if(isset($fc_item_in->field_fgs_ad_deadline_in_desc[LANGUAGE_NONE][0]['value'])){
+                    $desc_in = $fc_item_in->field_fgs_ad_deadline_in_desc[LANGUAGE_NONE][0]['value'];
+                }
+                echo '<p>' . $date_in . ' : ' . $desc_in . '</p>';
```
Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Add commenting

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 3]
```diff
@@ -95,3 +95,55 @@
...
+            foreach ($fc_item->field_fgs_ad_deadline_details_ca[LANGUAGE_NONE] as $key_ca => $value_ca) {
+                $fc_item_ca = field_collection_item_load($value_ca['value']);
+                if(isset($fc_item_ca->field_fgs_ad_deadline_ca_date[LANGUAGE_NONE][0]['value'])){
+                    //convert the date to desired format
+                    $date_ca = date("F j, Y" , strtotime($fc_item_ca->field_fgs_ad_deadline_ca_date[LANGUAGE_NONE][0]['value']));
+                }
+                if(isset($fc_item_ca->field_fgs_ad_deadline_ca_desc[LANGUAGE_NONE][0]['value'])){
+                    $desc_ca = $fc_item_ca->field_fgs_ad_deadline_ca_desc[LANGUAGE_NONE][0]['value'];
+                }
+                //print the admission deadline date and description about the this deadline
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Comment change: "print the admission deadline date and description about the this deadline" to "print the admission deadline date and description about this deadline"

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc [hunk 4]
```diff
@@ -118,6 +313,7 @@ function ucgy_fgs_field_default_field_bases() {
     'settings' => array(
       'allowed_values' => array(
         'Thesis based ' => ' Thesis based',
+        'Not Thesis based ' => ' Not Thesis based',
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc]: Recording for posterity, this is fixed in 912.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php (hunk 3 above)
Comment (nindusaini) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Thanks Dan, Change made.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc (hunk 4 above)
Comment (nindusaini) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.features.field_base.inc]: #912's AC is updated

**Comment by nindusaini** [comment]
Closing this because of conflicts, created a new one instead https://github.com/ucalgary/www.ucalgary/pull/532

**Pull request closed without merging**